test/test_torch.py

```diff
@@ -6607,6 +6607,20 @@ def test_serialization_filelike_uses_readinto(self):
         b = torch.load(data)
         self.assertTrue(data.was_called('readinto'))
 
+    def test_load_error_msg(self):
+        expected_err_msg = (".*You can only torch.load from a file that is seekable. " +
+                            "Please pre-load the data into a buffer like io.BytesIO and " +
+                            "try to load from it instead.")
+        if PY3:
+            import urllib.request
+            import io
+            resource = urllib.request.urlopen('https://download.pytorch.org/test_data/linear.pt')
```

Comment:
nit: this test could become a flaky test because it depends on a network resource. Is it possible to do something like
https://github.com/pytorch/pytorch/blob/master/test/common.py#L487-L511
where the test is skipped if the network resource is unavailable?